userland/examples/services/ble-env-sense/README.md

```diff
@@ -0,0 +1,71 @@
+BLE Environmental Sensing Service
+=================================
+
+This app uses BLE to provide the
+[environmental sensing service](https://www.bluetooth.com/specifications/assigned-numbers/environmental-sensing-service-characteristics)
+as an IPC service. This app configures the ESS profile for BLE, and then accepts
+updates for the sensor values as IPC messages.
+
+Example IPC Setup
+-----------------
+
+The service is identified as `org.tockos.services.ble-ess`. As an example, in C
+you might do:
+
+```c
+char buf[64] __attribute__((aligned(64)));
+
+static void ipc_callback(__attribute__ ((unused)) int pid,
+                         __attribute__ ((unused)) int len,
+                         __attribute__ ((unused)) int arg2,
+                         __attribute__ ((unused)) void* ud) {
+  printf("Updated BLE characteristic.\n");
+}
+
+int main() {
+  _svc_num = ipc_discover("org.tockos.services.ble-ess");
```

Comment:
This `_svc_num` variable need a type in this example. Plus maybe get rid of the underscore prefix:

```c
int svc_num = ipc_discover("org.tockos.services.ble-ess");
```